app/src/main/java/org/jellyfin/androidtv/util/profile/MediaCodecCapabilitiesTest.kt

```diff
@@ -33,6 +41,48 @@ class MediaCodecCapabilitiesTest {
 		CodecProfileLevel.AVCLevel4
 	)
 
+	private fun getHevcLevelString(profile: Int) : String {
+		val level = getDecoderLevel(MediaFormat.MIMETYPE_VIDEO_HEVC, profile)
+
+		when {
+			level < CodecProfileLevel.HEVCMainTierLevel1 -> return "0"
+			level < CodecProfileLevel.HEVCMainTierLevel2 -> return "30"
+			level < CodecProfileLevel.HEVCMainTierLevel21 -> return "60"
+			level < CodecProfileLevel.HEVCMainTierLevel3 -> return "63"
+			level < CodecProfileLevel.HEVCMainTierLevel31 -> return "90"
+			level < CodecProfileLevel.HEVCMainTierLevel4 -> return "93"
+			level < CodecProfileLevel.HEVCMainTierLevel41 -> return "120"
+			level < CodecProfileLevel.HEVCMainTierLevel5 -> return "123"
+			level < CodecProfileLevel.HEVCMainTierLevel51 -> return "150"
+			level < CodecProfileLevel.HEVCMainTierLevel52 -> return "153"
+			level < CodecProfileLevel.HEVCMainTierLevel6 -> return "156"
+			level < CodecProfileLevel.HEVCMainTierLevel61 -> return "180"
+			level < CodecProfileLevel.HEVCMainTierLevel62 -> return "183"
+			else -> return "186"
+		}
+	}
+
+	private fun getDecoderLevel(mime: String, profile: Int) : Int {
+		var maxLevel = 0
+
+		for (info in mediaCodecList.codecInfos) {
+			if (info.isEncoder) continue
+
+			try {
+				val capabilities = info.getCapabilitiesForType(mime)
+				for (profileLevel in capabilities.profileLevels) {
+					if (profileLevel.profile == profile) {
+						maxLevel = maxOf(maxLevel, profileLevel.level)
+					}
+				}
+			} catch (e: IllegalArgumentException) {
+				Timber.w(e)
```

Comment:
Add a message that explain why we ignore the exception

Reply:
It's copied from the below function, which has no comment either. I can add a comment here though.

Reply:
Yeah it can be really simple like `Timber.w("Unsupported codec $mime", e)`, but it helps a lot when tracing exceptions found in client logs.

Reply:
> Yeah it can be really simple like `Timber.w("Unsupported codec $mime", e)`, but it helps a lot when tracing exceptions found in client logs.

This exception isn't for an unsupported codec. It just happens for each codec we aren't currently looking for. I'd expect it to happen 20+ times for each time these two functions are called. 

They should both probably be debug and not warning messages.